Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -48,18 +49,19 @@ public function configDataProvider(): array
                 [
                     'namespace' => 'myapp',
                     'type' => 'redis',
-                    'ignored_routes' => ['prometheus_bundle_prometheus'],
                     'redis' => [
                         'host' => '127.0.0.1',
                         'port' => 6379,
                         'timeout' => 0.1,
-                        'read_timeout' => 10,
+                        'read_timeout' => '10',
```

Comment:
What is the reason of this change?

Reply:
I add assertSame to ensure the type of values. The read_timeout was already cast in string to avoid an issue with Redis::setOption()